service/publishing.gradle

```diff
@@ -0,0 +1,13 @@
+jib {
+	container {
+		filesModificationTime = ZonedDateTime.now().toString() // to prevent ui caching
+		jvmFlags = [
+				"-agentpath:/opt/cprof/profiler_java_agent.so=" +
+						"-cprof_service=bio.terra.martha" +
```

Comment:
don't forget to change this

Reply:
think I can remove this file altogether...don't think we'll be publishing any libraries

Reply:
this is for creating the image that is run on k8s